app/design/adminhtml/Magento/backend/web/js/theme.js

```diff
@@ -345,6 +346,25 @@ define('globalSearch', [
             this.input.on('focus.activateGlobalSearchForm', function () {
                 self.field.addClass(self.options.fieldActiveClass);
             });
+
+            $(document).keydown(function (event) {
```

Comment:
Could you switch to using 'on' with a `activateGlobalSearchForm` namespace, like this?
```suggestion
            $(document).on('keydown.activateGlobalSearchForm', function (event) {
```